Tests/SentryTests/Helper/SentryFileManagerTests.swift

```diff
@@ -139,6 +139,19 @@ class SentryFileManagerTests: XCTestCase {
         let actualData = envelopes[0].contents
         XCTAssertEqual(expectedData, actualData as Data)
     }
+
+    func testDeleteOldEnvelopes() throws {
+        let envelope = TestConstants.envelope
+        let path = sut.store(envelope)
+
+        try FileManager.default.setAttributes([FileAttributeKey.creationDate: Date(timeIntervalSince1970: 0)], ofItemAtPath: path)
```

Comment:
`m`: Please add one test case where the file is exactly 90 days old and doesn't get deleted and one where the file is 90.0001 days old, and gets deleted. This is testing the [boundary values of the equivalence classes](https://www.geeksforgeeks.org/equivalence-class-testing-next-date-problem/), which improves the test and minimizes off by one errors.

Reply:
Done